data-analytics/iot-nirvana/pipeline/run_oncloud.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+###
```

Comment:
Format the copyright header:
e.g.:
https://github.com/GoogleCloudPlatform/professional-services/blob/f6b875b098a07cd8e1198e2757b774236b3ff963/infrastructure/dns-sync/dns_sync/connect_with_dns_sync.sh

Reply:
Done in commit 2c396c90254c7b7cfcc5751d544636550b09c8b4